appveyor.yml

```diff
@@ -22,11 +22,10 @@ install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
 
+  # Use npm@6.10.3 to work-around issues w/ nyc@<15 on Windows
+  - npm install -g npm@6.10.3
```

Comment:
please prefer the if-based check; this won't work on node 4, for example.

Reply:
Node 4 is commented out in the Appveyor config. I usually only consider live code. But this branch is exploratory anyhow. I do not find the `npm` version pinning done here to be a satisfactory solution to the `nyc` problem (See https://github.com/benmosher/eslint-plugin-import/pull/1648#issuecomment-583602524).

Reply:
yes, but the intention is to uncomment it.

Fair that it's exploratory, but we were down to 1 failing test before you both started force pushing everywhere :-)

Reply:
As for local dev, the first step is to get CI working - all local dev by maintainers is done on a Mac.

Once CI is working, we can try to fix local Windows dev, for the rare contributor that uses it.

Reply:
> yes, but the intention is to uncomment it.

Understood 


> Fair that it's exploratory, but we were down to 1 failing test before you both started force pushing everywhere :-)

But test not running at all on Node 8 and 10.

Reply:
Again, first step is to get CI passing *somehow*.

We can uncomment node 10, 8, 6, 4, etc on appveyor in a future PR.

Reply:
> As for local dev, the first step is to get CI working - all local dev by maintainers is done on a Mac.

Understood